includes/object-cache.php

```diff
@@ -708,6 +709,7 @@ protected function connect_using_phpredis( $parameters ) {
                     $parameters['scheme'],
                     str_replace( 'tls://', '', $parameters['host'] )
                 );
+                $args['ssl'] = $parameters['ssl'];
             }
 
```

Comment:
The `WP_REDIS_TLS` constant needs to be shown in the diagnostics.

Reply:
I already responded to this. As far as I can see it *is* shown in the diagnostics, because you do `$this->diagnostics += $args;` on line 743.

Am I misunderstanding?

Reply:
Yes, the constant itself needs to be listed as well.

Reply:
Ahh! includes/diagnostics.php:68?